### PR TITLE
Included `add` to O0's quant op type list

### PR DESCRIPTION
**Pull request by yiliu30**
## Type of Change

Enhancement
API changed or not: None

## Description

If smooth quantization is enabled, include the `add` to O0's quantization op list.

## Expected Behavior & Potential Risk

**Quant type list change**:
```diff
- ['conv', 'matmul', 'linear']
+ ['conv', 'matmul', 'linear', 'add'] # If smooth_quant is True
```
Also, add [`bmm`](https://pytorch.org/docs/stable/generated/torch.bmm.html) to handle the IPEX batch matrix-matrix product 

**Potential change**:
- The tuning count of some models may increase by one.
- Some models may only quantize the `add`.

## How has this PR been tested?

Pre-CI and Extension test

## Dependency Change?

None

### CODE REVIEW

**Comment by yiliu30** [comment]
[Extension test for gtp-j-68](https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/lastSuccessfulBuild/artifact/report.html) with `quant_level=0`
All `Linear` and `Matmul` operations are int8, while all `add` are fp32.

File: examples/pytorch/nlp/huggingface_models/language-modeling/quantization/ptq_static/ipex/smooth_quant/run_tuning.sh [hunk 1]
```diff
@@ -28,6 +28,10 @@ function run_tuning {
     if [[ ${input_model} == *"gpt-j-6B" ]]; then
         ext_cmd="--alpha auto --fallback_add"
     fi
+    # disable manually  fallback add
```
Comment (hshen14) [inline examples/pytorch/nlp/huggingface_models/language-modeling/quantization/ptq_static/ipex/smooth_quant/run_tuning.sh:31]: remove comments

File: neural_compressor/strategy/conservative.py [hunk 2]
```diff
@@ -76,9 +76,18 @@ def __init__(self,
                          q_hooks=q_hooks)
         logger.info(f"*** Initialize conservative tuning")
         self.acc_meet_flag = False
-        self.quant_op_type_lst = ['conv', 'matmul', 'linear']
+        self.quant_op_type_lst = ['conv', 'matmul', 'bmm', 'linear']
```
Comment (hshen14) [inline neural_compressor/strategy/conservative.py:79]: why you add bmm？

**Review by changwangss — APPROVED** [review]

File: examples/pytorch/nlp/huggingface_models/language-modeling/quantization/ptq_static/ipex/smooth_quant/run_tuning.sh (hunk 1 above)
Comment (yiliu30) [reply examples/pytorch/nlp/huggingface_models/language-modeling/quantization/ptq_static/ipex/smooth_quant/run_tuning.sh]: Thx, removed by [e4d8090](https://github.com/intel/neural-compressor/pull/1090/commits/e4d80904787b48ee735ba353ee7a00a9af5274ec)

**Review by Kaihui-intel — APPROVED** [review]

**Pull request merged**